HAL/pico/src/comms/NintendoSwitchBackend.cpp

```diff
@@ -100,43 +100,51 @@ void NintendoSwitchBackend::RegisterDescriptor() {
 }
 
 void NintendoSwitchBackend::SendReport() {
-    ScanInputs(InputScanSpeed::SLOW);
-    ScanInputs(InputScanSpeed::MEDIUM);
+  ScanInputs(InputScanSpeed::SLOW);
+  ScanInputs(InputScanSpeed::MEDIUM);
 
-    while (!TUCompositeHID::_usb_hid.ready()) {
-        tight_loop_contents();
-    }
+  while (!TUCompositeHID::_usb_hid.ready()) {
+      tight_loop_contents();
+  }
+
+  ScanInputs(InputScanSpeed::FAST);
+
+  UpdateOutputs();
+
+  // Digital outputs
+  _report.y = _outputs.y;
+  _report.b = _outputs.b;
+  _report.a = _outputs.a;
+  _report.x = _outputs.x;
+
+  // TODO: This flips R1/R2 and L1/L2 but the game actually sees b0xx L and R as the triggers.
+  // Maybe the pinout needs to change? Unsure.
+  _report.l = _outputs.triggerLDigital;
+  _report.r = _outputs.triggerRDigital;
+  _report.zl = _outputs.buttonL;
+  _report.zr = _outputs.buttonR;
```

Comment:
This seems consistent with 3rd party "Pro Controllers", these are always flipped.
Not sure if we could actually fix this in a better way, but I expect not.
Doing this allows for us to reset training mode.